### PR TITLE
Improve checks on Nextcloud Talk start

### PR DESCRIPTION
**Pull request by bertrandda**
### Pull Request check-list

To ensure your Pull Request can be accepted as fast as possible, make sure to review and check all of these items:

- [x] If your changes affects code, did your write the tests?
- [x] Are tests passing? (`npm test` on both front/server)
- [x] Is the linter passing? (`npm run eslint` on both front/server)
- [x] Did you run prettier? (`npm run prettier` on both front/server)
- [x] Did you test this pull request in real life? With real devices? If this development is a big feature or a new service, we recommend that you provide a Docker image to [the community](https://community.gladysassistant.com/) for testing before merging.

NOTE: these things are not required to open a PR and can be done afterwards / while the PR is open.

### Description of change

Please provide a description of the change here. It's always best with screenshots, so don't hesitate to add some!

Add checks to start NC Talk service only if it is correctly configured
Also increase interval after a request has failed

### CODE REVIEW

File: server/services/nextcloud-talk/lib/bot/bot.startPolling.js [hunk 1]
```diff
@@ -22,7 +22,48 @@ async function startPolling(userId) {
     userId,
   );
 
+  if (
+    !this.bots[userId].NEXTCLOUD_URL ||
+    !this.bots[userId].NEXTCLOUD_BOT_USERNAME ||
+    !this.bots[userId].NEXTCLOUD_BOT_PASSWORD
+  ) {
+    logger.debug(`Cancel Nextcloud Talk bot starting for user: ${userId}, missing configuration`);
+    delete this.bots[userId];
```
Comment (Pierre-Gilles) [inline server/services/nextcloud-talk/lib/bot/bot.startPolling.js:31]: I don't think this will remove the listener, you need to call removeListener

**Review by Pierre-Gilles — COMMENTED** [review]
Thanks for the quick fix, I've wrote a feedback about the message listener!

File: server/services/nextcloud-talk/lib/bot/bot.startPolling.js (hunk 1 above)
Comment (bertrandda) [reply server/services/nextcloud-talk/lib/bot/bot.startPolling.js:31]: That's why I moved the addListener at the end of this function
https://github.com/GladysAssistant/Gladys/pull/1832/files#diff-c305aedfe417914247d3eeaa16a6c8d999e24c20c0c8f3a12ed3e6abbff83ad1R66

Comment (Pierre-Gilles) [reply server/services/nextcloud-talk/lib/bot/bot.startPolling.js:31]: But if startPolling is called again, will it create a second listener?

Comment (bertrandda) [reply server/services/nextcloud-talk/lib/bot/bot.startPolling.js:31]: Indeed, did not think about that case, should be better now

Comment (Pierre-Gilles) [reply server/services/nextcloud-talk/lib/bot/bot.startPolling.js:31]: Thanks for fixing!

**Review by Pierre-Gilles — APPROVED** [review]
Thanks for this PR ! 🙏

**Pull request merged**